solr/conf/schema.xml

```diff
@@ -194,14 +202,30 @@
     <dynamicField name="*_llsi" type="location" stored="true" indexed="true" multiValued="false"/>
     <dynamicField name="*_llsim" type="location" stored="true" indexed="true" multiValued="true"/>
 
-    <dynamicField name="*_srpt" type="location_rpt" stored="true" indexed="true" multiValued="true" />
+    <dynamicField name="*_srpt" type="location_rpt" stored="true" indexed="true" multiValued="true"/>
 
     <!-- suggest and spelling -->
-    <dynamicField name="*spell" type="textSpell" indexed="true" stored="false" multiValued="true" />
-    <dynamicField name="*suggest" type="textSuggest" indexed="true" stored="false" multiValued="true" />
+    <dynamicField name="*spell" type="textSpell" indexed="true" stored="false" multiValued="true"/>
+    <dynamicField name="*suggest" type="textSuggest" indexed="true" stored="false" multiValued="true"/>
 
     <!-- JSON -->
-    <dynamicField name="*_struct" type="string" stored="true" indexed="false" multiValued="true" omitNorms="true" />
+    <dynamicField name="*_struct" type="string" stored="true" indexed="false" multiValued="true" omitNorms="true"/>
+
+    <!-- docValue tests -->
+    <!-- Facets -->
+    <field name="access_facet"        type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="all_authors_facet"   type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="format"              type="string" docValues="true" stored="false" multiValued="true"/>
+    <field name="genre_facet"         type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="genre_full_facet"    type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="language_facet"      type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="lc_1letter_facet"    type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="pub_date_facet"      type="string" docValues="true" stored="false" multiValued="true" />
+    <field name="subject_topic_facet" type="string" docValues="true" stored="false" multiValued="true" />
+
+    <!-- Sorts -->
+    <field name="pub_date_sort"       type="string" docValues="true" stored="false" multiValued="true" />
```

Comment:
@cdmo I think `pub_date_sort_itsi` had  docValues enabled already.  [`type: tint`](https://github.com/psu-libraries/psulib_blacklight/blob/8db6bcc9d5be9cc8c3a5c9b581dfb3b79c4bd7db/solr/conf/schema.xml#L98) is defined as  [`<fieldType name="tint" class="solr.IntPointField" docValues="true"/>`](https://github.com/psu-libraries/psulib_blacklight/blob/8db6bcc9d5be9cc8c3a5c9b581dfb3b79c4bd7db/solr/conf/schema.xml#L287).

Reply:
Yup, that's right, good catch thanks!